jalib/jalloc.h

```diff
@@ -22,35 +22,70 @@
 #ifndef JALLOC_H
 #define JALLOC_H
 
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
 #define JALIB_ALLOCATOR
+#define JALLOC_DEBUG
 
-#include <stdlib.h>
+static constexpr size_t headerSizeInBytes = sizeof(size_t);
+static constexpr size_t footerSizeInBytes = sizeof(size_t);
```

Comment:
How about:
  `footerSizeInBytes` -> `headerDebugSizeInBytes`
and also in the code below?